ext/greenwich/scss/_greenwich.scss

```diff
@@ -566,7 +566,7 @@ $popover-arrow-color: $popover-bg;
 //** Popover outer arrow width
 $popover-arrow-outer-width: ($popover-arrow-width + 1);
 //** Popover outer arrow color
-$popover-arrow-outer-color: fadein($popover-border-color, 5%);
+$popover-arrow-outer-color: fadein($popover-border-color, .5%);
```

Comment:
I'm just wondering if this is supposed to represent an opacity between 0 and 1, should it be `.05`, without the `%`? If it's supposed to be a percent, why doesn't `5%` as-is work? Both `.5%` and `.05` prevent the error when I run composer, I just don't know where this ends up getting used to check what the result is.

Reply:
Good question. It looks like the original line from the Bootstrap3 source code is `fade_in($popover-border-color, 0.05);` and changing it to that compiles for me without error. Also PhpStorm was giving a red-underline to `fadein` and is happier with `fade_in`. I've updated this PR with that value.

Reply:
Compiles ok for me too.

Reply:
The error stack has this row:
ScssPhp\ScssPhp\Node\Number->valueInRange(0, 1, 'amount')
which suggests that the % is being ignored and that the number should be 0.05 to represent 5%
I can see that coleman has fixed his commit and pushed it, so I'm just leaving this here for posterity ...

Reply:
The API of this function expects a number between 0 and 1: https://sass-lang.com/documentation/modules/color#opacify
Our implementation is consistent with dart-sass in ignoring the unit instead of enforcing unitless numbers.

And the name should indeed be `fade-in` (or `fade_in` as underscores and dashes are equivalent in Sass function names), not `fadein`. `fadein` works in scssphp only due to the internal implementation of core functions in scssphp and the fact that PHP method names are case-insensitive (and we convert names to camelCase to find the method to call).

Reply:
Thanks for confirming!